### ISSUE TITLE
£350 budget for standing desks

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/london-money#32]
As requested by @JMurphyWeb.

https://github.com/foundersandcoders/london-programme/issues/249

Any objections?

**Comment by sofer** [comment]
I don't hear any objections. 

@JMurphyWeb I think we have a £350 budget approved :tada:

**Comment by markwilliamfirth** [comment]
I think the accounts still require being finalised before large (especially enhancement) purchases are made - at our last meeting we only had rough numbers?

**Comment by sofer** [comment]
@markwilliamfirth oh gosh. You are reminding me of all the things we still need to do. I have just checked Xero and we have a net operating profit this month of £13,350. This does not include significant sums due to be raised in the next week for recruitment (and also payments due from dwyl). We certainly have the funds for this and we have an incipient and urgent need for more space. Could we please take a lenient view of the state of the budget and authorise this expenditure otherwise I fear the opportunity will pass (Jack has offered to do the work this weekend)?

I'll try to find the time to prepare a more complete budget this week.

**Comment by JMurphyWeb** [comment]
Im figuring out my schedule over the next couple days. Either this weekend or next I can definitely commit a day. Do need to know soon from your end though.

@jackcarlisle is keen to assist if he is available. Would be great to have anyone else help out.

**Comment by markwilliamfirth** [comment]
Ok just thought I'd voice the concern! We do really need an accurate view of expenditure as well as income in order to set budgets in future though

**Comment by jsms90** [comment]
I like standing desks, but I wouldn't consider this a priority for us. Not when compared to all the other things that FAC could be spending its money on.

I won't stand in the way of this happening, since you are the one across the accounts atm @sofer, and you are happy with this. But for as long as our accounts stay in their current state, I feel incapable of making informed decisions.

If this were an actual vote, I would abstain.

**Comment by markwilliamfirth** [comment]
Agreed - we can't make informed decisions without the data. The course facilitator was a different situation because:

- it was an urgent priority
- it fell within budget estimations
- it had a termination clause to protect future expenditure
- it was a necessity for FAC to continue

Do we know when the accounts will be updated?

**Comment by sohilpandya** [comment]
@markwilliamfirth, sorry to butt in but I would like to know the thinking behind the last point that you've made:
> it was a necessity for FAC to continue

FAC existed fine and produced a wonderful group of alumni all the way up to FAC9 without a course facilitator, so would like to know why it is a necessity for FAC to continue.

I don't want to pollute this issue, will create a new one where you can potentially respond. 👍

**Comment by markwilliamfirth** [comment]
@sohilpandya should have said:

*to continue to deliver a satisfactory course

There is a history of some of the various roles of the course facilitator not being actioned during particular cohorts and a clear correlation with poorer cohort outcomes. It's been shown that a consistent presence delivering a role that covers these responsibilities positively increases course satisfaction and outcomes. These outcomes include the number of participants that successfully complete the course, community and transitions into paid employment (with F&C taking a hiring fee).

**Comment by sohilpandya** [comment]
@markwilliamfirth 

Agreed that certain roles have taken a backseat in the past and this role does help improve that aspect 👍 and @jsms90 has done a great job of keeping FAC10 happy and comforted for the first 8 weeks. Though personally, I am not a fan of it as I feel it should still be more self-organising as it has been in the past.

- I'd probably argue that the reason the people are transitioning into paid employment is due to @freelancedad doing a great job and little to do with a course facilitator for FAC10. (IMO Course facilitation has nothing to do with employment)
- We can't have a clear correlation when only one cohort has had a course coordinator, we need more for that statistic to be proven correct which I'm sure it will.
- FAC6 - 16, FAC7 - 15, FAC8 - 16, FAC9 - 16, FAC10 - 16. Again, need to wait a little longer for the stat that more participants successfully complete the course. 

Potentially carry on this conversation here - https://github.com/foundersandcoders/london-programme/issues/257

**Comment by markwilliamfirth** [comment]
>I'd probably argue that the reason the people are transitioning into paid employment is due to @freelancedad doing a great job and little to do with a course facilitator for FAC10

True, although the course facilitator does this indirectly. Some cohorts have only finished with 9 or 10 people coming out of the course at the end - this affects how many people can be placed

>We can't have a clear correlation when only one cohort has had a course coordinator, we need more for that statistic to be proven correct which I'm sure it will.

This is based on experience and anecdotal evidence - agreed that [stats should be being measured](https://github.com/foundersandcoders/international/issues/63#issuecomment-309313552). Also some of the responsibilities of the CF role have been taken on by other people in some cohorts and not in others. A notable example of one of these responsibilities is the importance of a continuous figure in pastoral care and maintaining cohort cohesion.

**Comment by rebeccaradding** [comment]
@sohilpandya @markwilliamfirth from what I understand, @sofer @iteles held many of CF responsibilities prior to FAC8 - so while these cohorts may certainly have been more self-organising, they weren't left completely alone (though please correct me if I'm wrong). 

During FAC8, @iteles stepped back and @sofer did quite a bit of traveling, making apparent the need for Dan to share this work with others more explicitly.

**Comment by sohilpandya** [comment]
@markwilliamfirth agreed that at times not all roles were being picked up. 

@rebeccaradding you are correct, @sofer was heavily invested in catch ups (CF role), getting graduate projects(not a CF Role) along with many other things from what I remember during previous cohorts.

**Comment by JMurphyWeb** [comment]
@sofer I can take the hit on these.. It can go toward my FAC contribution?

@jackcarlisle kindly putting himself forward to assist. Would be great to have a few more. Sunday morning. 10am - 2pm.

**Comment by markwilliamfirth** [comment]
@JMurphyWeb if you go ahead make sure you keep the receipt - it's [currently under proposal](https://github.com/foundersandcoders/london-money/issues/35) so there may be a chance you can get reimbursed, if not fully maybe partially. It's best for FAC contributions to come in the form of donations, as VAT on purchases like this can be claimed back by FAC.

**Comment by sofer** [comment]
@JMurphyWeb I am keen for this work to go ahead Sunday, particularly if you are prepared to do it regardless of the outcome of our budget machinations.

Please do confer with other users of the space to make sure that everyone else is happy with your plan. 

I do hope you get more volunteers.

And thank you ❤️

**Comment by JMurphyWeb** [comment]
@sofer i'm planning on making them free standing so anything we put in place can be moved

**Comment by JMurphyWeb** [comment]
I visited to wickes today only to find out they didn't have the saw that we need in stock. @jackcarlisle and I are going to work on a design this week and will update this issue when we have a date in mind for putting something together.
